esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2214,8 +2233,35 @@ bool CFileSprayEx::onReplicate(IEspContext &context, IEspReplicate &req, IEspRep
     return true;
 }
 
+void CFileSprayEx::getDropZoneInfoByDestPlane(double clientVersion, const char* destGroup, const char* destFileIn, StringBuffer& destFileOut, StringBuffer& umask)
+{
+    const bool isDestAbsolutePath = isAbsolutePath(destFileIn);
+    Owned<IPropertyTreeIterator> planes = getDropZonePlanesIterator(destGroup);
```

Comment:
Is destGroup required?  If so, isn't there only a single entry for a given plane?

Reply:
destGroup (now renamed to destPlane) is required.  Yes, there is only ever one matching.  I'll look into removing the iterator.